### ISSUE TITLE
'key' of 'val_data_loader' in visualisation_config.yaml should be replaced.

### ISSUE THREAD

**Issue by Akhil-Raj** [lyft/l5kit#293]
As of version 1.3, config file 'visualisation_config.yaml' has the value of 'key' key in the dictionary 'val_data_loader' set to 'scenes/sample.zarr'. But that leads to a problem that 'AgentDataset' is returning an empty dataset.

Changing it the value to 'scenes/validate.zarr' solved the problem.

**Comment by lucabergamini** [comment]
`sample.zarr` is a valid split for the dataset so the code should run with it. Is it possible that you have not downloaded that split in your local folder maybe?

**Comment by Akhil-Raj** [comment]
Yes, you are right. I re-downloaded the sample.tar and it worked.
